packages/components/progress-bar/src/progress-bar.ts

```diff
@@ -91,10 +109,8 @@ export class ProgressBar extends ScopedElementsMixin(LitElement) {
       </div>
       <div id="helper" class="helper">
         <slot></slot>
-        <span id="live" aria-live="polite" aria-busy=${ifDefined(this.indeterminate)}>
-          ${msg('state')} ${this.variant ? html`${this.#getLocalizedVariant()}` : html`${msg('active')}`}
-          <!-- We want '%' to be read every time the value changes. -->
-          <span aria-live="polite" aria-atomic="true">${this.value}%</span>
+        <span id="live" aria-busy=${ifDefined(this.indeterminate)}>
+          ${msg('state')}: ${this.variant ? html`${this.#getLocalizedVariant()}` : html`${msg('active')}`}
```

Comment:
Right now I'm not getting an information anymore that the state has changed (variant)?
In this example it should be said that the state has changed into `success`? Right now I'm getting only information that it's 100%.
<img width="1386" alt="image" src="https://github.com/user-attachments/assets/fd3f3e24-70fa-4adf-8adb-a2be9e226f66">

Reply:
If an error occured while the progressbar is filling up, shouldn't there be more information than just the state changing? An inline message for example, with more information on what when wrong? If we then also announce the change in state you have the information twice.
I've added the state of the element when it's 100%, but I don't really see an added value in having it announced when the state changes; that will not be enough information.

Reply:
I've updated the documentation for the progressbar to indicate what is does and does not do and what we can't possibly do in the component (because we don't know the reason behind things happening) and what the dev need to do in their cfa.